resources/assets/styles/layouts/hero.scss

```diff
@@ -0,0 +1,169 @@
+.hero {
+  @include cssLock("margin-top", 360, 1920, ($gutter_num * 7.8, $gutter_num * 8));
+  @include cssLock("max-height", 360, 1920, ($gutter_num * 50, $gutter_num *100));
+
+  position: relative;
+
+  .flickity-viewport {
+    @include cssLock("max-height", 360, 1920, ($gutter_num * 50, $gutter_num *100));
+  }
+
+  &__cell {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+  }
+
+  &__overlay {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: color(primary);
+    opacity: 0.3;
+    z-index: -1;
+  }
+
+  &__image {
+    position: absolute;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+    z-index: -2;
+    object-fit: cover;
+    height: 100%;
+  }
+
+  &__content {
+    @include cssLock("padding-top", 360, 1920, ($gutter_num * 7.8, $gutter_num * 8));
+    @include cssLock("padding-bottom", 360, 1920, ($gutter_num * 7.8, $gutter_num * 8));
+  }
+
+  &__slogan {
+    @extend .headline;
+
+    @include cssLock("padding-top", 360, 1920, ($gutter_num*6, $gutter_num * 19));
+    @include cssLock(
+      "margin-bottom",
+      360,
+      1920,
+      ($gutter_num, $gutter_num * 4)
+    );
+
+    display: flex;
+    flex-direction: column;
+    text-transform: uppercase;
+    color: color(primary);
+  }
+
+  &__label {
+    @extend .subtitle;
+
+    @include cssLock("padding-left", 360, 1920, ($gutter_num, $gutter_num * 2));
+    @include cssLock(
+      "padding-right",
+      360,
+      1920,
+      ($gutter_num, $gutter_num * 2)
+    );
+    @include cssLock(
+      "padding-top",
+      360,
+      1920,
+      ($gutter_num, $gutter_num * 1.7)
+    );
+    @include cssLock(
+      "padding-bottom",
+      360,
+      1920,
+      ($gutter_num, $gutter_num * 1.7)
+    );
+    @include cssLock(
+      "margin-bottom",
+      360,
+      1920,
+      ($gutter_num, $gutter_num * 1.7)
+    );
+    @include cssLock("width", 360, 1920, ($gutter_num * 10, $gutter_num * 17));
+
+    display: block;
+    background-color: color(special);
+    color: color(secondary);
+    text-transform: none;
+  }
+
+  &__description {
+    @include cssLock(
+      "margin-top",
+      360,
+      1920,
+      ($gutter_num * 1.7, $gutter_num * 6.25)
+    );
+    @include cssLock(
+      "margin-bottom",
+      360,
+      1920,
+      ($gutter_num * 2.5, $gutter_num * 6.25)
+    );
+
+    width: 80%;
```

Comment:
dla mnie te width:80% to magic number :) 
już lepiej zastosować w tym miejscu max-width oreślony np na około 600px